src/components/Dropdown/Dropdown.js

```diff
@@ -0,0 +1,69 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Select from 'react-select';
+
+import { connectTheme } from '../../utils';
+
+/*
+ * See full API here:
+ * https://github.com/JedWatson/react-select/#usage
+ */
+
+// options prop populates Select
+// onChange({ label, value }, { action, option }) => void
+
+class Dropdown extends PureComponent {
+  static get propTypes() {
+    return {
+      theme: PropTypes.object,
+      style: PropTypes.object,
+      options: PropTypes.array
+    };
+  }
+
+  // options shape is { value, label }
+  // if options array[0], assume proper shape
+  // otherwise, map to { value: element, label: element }
+  formatOptions(options = []) {
+    // assume all elements same type
+    const t = typeof options[0];
+    if (t === 'string' || t === 'number') {
+      return options.map(el => ({ label: `${el}`, value: `${el}` }));
+    }
+    return options;
+  }
+
+  customStyles(theme) {
+    const { themeVariables = {} } = theme;
+    const { themeColors = {} } = themeVariables;
+
```

Comment:
This is where the theme can interact with the library. I'm not exactly sure which parts of the theme to style the dropdown with